src/lib/containers/CardContainer.tsx

```diff
@@ -59,7 +59,7 @@ export const CardContainer = (props: CardContainerProps) => {
       case OBSERVATION_CARD:
         return <ObservationCard {...props} />
       default:
-        return <div /> // this should never happen
+        return <div key={props.key} /> // this should never happen
```

Comment:
I think it does happen in the test. For now, give it a key to suppress a warning.

Reply:
Please remove or update the comment if it's incorrect